koku/reporting/provider/gcp/models.py

```diff
@@ -81,3 +101,137 @@ class Meta:
+    usage_end = models.DateField(null=True)
+    tags = JSONField(null=True)
+    usage_type = models.CharField(max_length=50, null=True)
+    region = models.CharField(max_length=256, null=True, blank=True)
+    cost = models.DecimalField(max_digits=24, decimal_places=9, null=True, blank=True)
+    currency = models.CharField(max_length=256, null=True, blank=True)
+    conversion_rate = models.CharField(max_length=256, null=True, blank=True)
+    usage_amount = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    usage_unit = models.CharField(max_length=256, null=True, blank=True)
+    usage_to_pricing_units = models.DecimalField(max_digits=24, decimal_places=9, null=True)
+    usage_pricing_unit = models.CharField(max_length=256, null=True, blank=True)
+    invoice_month = models.CharField(max_length=256, null=True, blank=True)
+    tax_type = models.CharField(max_length=256, null=True, blank=True)
+
+
+class GCPCostEntryLineItemDailySummary(models.Model):
+    """A daily aggregation of line items.
+
+    This table is aggregated by service, and does not
+    have a breakdown by resource or tags. The contents of this table
+    should be considered ephemeral. It will be regularly deleted from
+    and repopulated.
+
+    """
+
+    class Meta:
+        """Meta for GCPCostEntryLineItemDailySummary."""
+
+        managed = False  # for partitioning
+
+        db_table = "reporting_gcpcostentrylineitem_daily_summary"
+        indexes = [
+            models.Index(fields=["usage_start"], name="gcp_summary_usage_start_idx"),
+            models.Index(fields=["instance_type"], name="gcp_summary_instance_type_idx"),
+            GinIndex(fields=["tags"], name="gcp_tags_idx"),
+            models.Index(fields=["project_id"], name="gcp_summary_project_id_idx"),
+            models.Index(fields=["project_name"], name="gcp_summary_project_name_idx"),
+            models.Index(fields=["service_id"], name="gcp_summary_service_id_idx"),
+            models.Index(fields=["service_alias"], name="gcp_summary_service_alias_idx"),
+        ]
+
+    uuid = models.UUIDField(primary_key=True)
+
+    cost_entry_bill = models.ForeignKey(GCPCostEntryBill, on_delete=models.CASCADE)
+
+    # The following fields are used for grouping
+    account_id = models.CharField(max_length=20)
+    project_id = models.CharField(max_length=256)
+    project_name = models.CharField(max_length=256)
+    service_id = models.CharField(max_length=256, null=True)
+    service_alias = models.CharField(max_length=256, null=True, blank=True)
+    sku_id = models.CharField(max_length=256, null=True)
+    sku_alias = models.CharField(max_length=256, null=True)
+    usage_start = models.DateField(null=False)
+    usage_end = models.DateField(null=True)
+    region = models.CharField(max_length=50, null=True)
+    instance_type = models.CharField(max_length=50, null=True)
+    usage_to_pricing_units = models.DecimalField(max_digits=24, decimal_places=9, null=True)
```

Comment:
Hate to come back with more suggestions, but after thinking about this a bit more. Let's remove `usage_to_pricing_units` in the daily summary table and just populate `usage_amount` with that value from the daily table.